opentelekomcloud/resource_opentelekomcloud_lb_listener_v2_test.go

```diff
@@ -31,7 +31,36 @@ func TestAccLBV2Listener_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						"opentelekomcloud_lb_listener_v2.listener_1", "name", "listener_1_updated"),
 					/* resource.TestCheckResourceAttr(
-					"opentelekomcloud_lb_listener_v2.listener_1", "connection_limit", "100"), */
+					   "opentelekomcloud_lb_listener_v2.listener_1", "connection_limit", "100"), */
+				),
+			},
+			{
+				Config: TestAccLBV2ListenerConfig_http2,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckLBV2ListenerExists("opentelekomcloud_lb_listener_v2.listener_tls", &listener),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "http2_enable", "true"),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "admin_state_up", "true"),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "tls_ciphers_policy", "tls-1-1"),
+					resource.TestCheckResourceAttrSet(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "default_tls_container_ref"),
+				),
+			},
+			{
+				Config: TestAccLBV2ListenerConfig_update_http2,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "name", "listener_tls_updated"),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "http2_enable", "false"),
+					resource.TestCheckResourceAttr(
+						"opentelekomcloud_lb_listener_v2.listener_tls", "admin_state_up", "true"),
```

Comment:
Can you please add `admin_state_up` `false` to some test step

Reply:
Tried this out and found out that at the moment, only true is supported. Another chinese OTC stupidity.

Reply:
*yet*